hms-app/src/components/card-details/HackathonDetails.tsx

```diff
@@ -59,15 +59,18 @@ export default function HackathonDetails(props: IProps) {
   const [hackathonData, setHackathonData] = useState({} as Hackathon)
   const [ideaData, setIdeaData] = useState<Idea>()
   const [relevantIdeaList, setRelevantIdeaList] = useState([] as Idea[])
-  const [value, onChange] = useState(hackathonData.description)
   const [votingOpened, setVotingOpened] = useState<boolean>(false)
 
+  const initialValue =
```

Comment:
naming it something like `descriptionHint` makes it a bit more obvious to it's purpose